### ISSUE TITLE
[Question] CollapsiblePanel component - aria-expanded in the wrong place.

### ISSUE THREAD

**Issue by EmmanuelDemey** [got5/KAWWA#111]
the aria-expanded attribute should be in the "a" element If not, the Jaws/NVDA tool will not say "expanded..." when the focus is on the link.
